draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -692,8 +692,8 @@
    </t>
 
    <t>
-	dCDN SHOULD provide continuous update of the trigger processing progress, by responding with updated trigger resource representation 
-	to subsequent uCDN requests sent to the created trigger URL. 
+	dCDN SHOULD provide continuous updates of the trigger processing progress, by responding with updated trigger resource representations 
```

Comment:
The comma between "progress" and "by" is unnecessary as well.

Reply:
fixed